packages/api/src/controllers/stream.ts

```diff
@@ -841,12 +842,21 @@ app.get("/:id", authorizer({}), async (req, res) => {
 
 // returns stream by steamKey
```

Comment:
mind fixing this comment while you're here 😆 
```suggestion
// returns stream by playbackId
```